packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -151,14 +151,18 @@
                     data-test-id="user-info-icon"
                     v-on="isBelowMid ? null : { mouseover: openUserMenu, mouseleave: closeUserMenu }"
                     @keyup.esc="closeUserMenu">
-                    <a
+                    <button
```

Comment:
Could you use ghost f-button in this case? There is `hasIcon=leading` prop so it should look like you need, I think

Reply:
It's a good idea but there are quite a lot of differences with the styling at the minute, so I would need to put in lots of overrides which does defeat the point slightly. Perhaps once we have new header designs it might be more fitting?